lib/src/component_declaration/component_base.dart

```diff
@@ -441,9 +453,7 @@ abstract class StateMapViewMixin {
 ///
 /// For use by concrete [UiProps] and [UiState] implementations (either generated or manual),
 /// and thus must remain public.
-abstract class MapViewMixin<K, V> {
-  Map<K, V> get _map;
```

Comment:
Unfortunately, these type parameters had to be dropped; parameterizing the base class (and omitting those parameters in PropsMapViewMixin) results in this issue still being present.

We shouldn't really miss these parameters, though, anyway.